### ISSUE TITLE
Rails 5.0.0/Ruby 2.3.1 can't modify frozen Array (RuntimeError)

### ISSUE THREAD

**Issue by nullset2** [ForestAdmin/forest-rails#65]
I've ran into this issue only when running the app in production, not in development.

My Gemfile:

```
source 'https://rubygems.org'

ruby '2.3.1'

gem 'rails', '~> 5.0.0'
gem 'pg', '~> 0.18'
gem 'puma', '~> 3.0'
gem 'devise_token_auth', github:"lynndylanhurley/devise_token_auth", branch: "master"
gem 'rack-cors', require: 'rack/cors'
gem 'omniauth'
gem 'omniauth-facebook'
gem 'omniauth-google-oauth2'
gem 'figaro'
gem 'jbuilder'
gem 'pry-rails'
gem 'validates_timeliness'
gem 'paperclip'
gem 'aws-sdk', '>= 2.0.34'
gem 'geocoder'
gem 'ransack'
gem 'apipie-rails'
gem 'forest_liana'

group :development, :test do
  gem 'byebug', platform: :mri
end

group :development do
  gem 'listen', '~> 3.0.5'
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
end

gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

My stacktrace:

```
=> Booting Puma
=> Rails 5.0.0 application starting in production on http://0.0.0.0:3000
=> Run `rails server -h` for more startup options
Exiting
/Users/alfredo/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0/lib/action_dispatch/middleware/stack.rb:74:in `insert': can't modify frozen Array (RuntimeError)
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0/lib/action_dispatch/middleware/stack.rb:74:in `insert'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/forest_liana-1.3.50/lib/forest_liana/engine.rb:22:in `configure_forest_cors'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/railtie.rb:193:in `public_send'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/railtie.rb:193:in `method_missing'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/forest_liana-1.3.50/lib/forest_liana/engine.rb:36:in `block in <class:Engine>'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/activesupport-5.0.0/lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/activesupport-5.0.0/lib/active_support/lazy_load_hooks.rb:45:in `block in run_load_hooks'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/activesupport-5.0.0/lib/active_support/lazy_load_hooks.rb:44:in `each'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/activesupport-5.0.0/lib/active_support/lazy_load_hooks.rb:44:in `run_load_hooks'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/application/finisher.rb:65:in `block in <module:Finisher>'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/initializable.rb:30:in `instance_exec'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/initializable.rb:30:in `run'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:228:in `block in tsort_each'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:347:in `each'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:347:in `call'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:347:in `each_strongly_connected_component'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:226:in `tsort_each'
	from /Users/alfredo/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/tsort.rb:205:in `tsort_each'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/initializable.rb:54:in `run_initializers'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/application.rb:352:in `initialize!'
	from /Users/alfredo/Documents/workspace/zulu-rails/config/environment.rb:5:in `<top (required)>'
	from /Users/alfredo/Documents/workspace/zulu-rails/config.ru:3:in `require_relative'
	from /Users/alfredo/Documents/workspace/zulu-rails/config.ru:3:in `block in <main>'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/builder.rb:55:in `instance_eval'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/builder.rb:55:in `initialize'
	from /Users/alfredo/Documents/workspace/zulu-rails/config.ru:in `new'
	from /Users/alfredo/Documents/workspace/zulu-rails/config.ru:in `<main>'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/builder.rb:49:in `eval'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/builder.rb:49:in `new_from_string'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/builder.rb:40:in `parse_file'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/server.rb:318:in `build_app_and_options_from_config'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/server.rb:218:in `app'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/commands/server.rb:59:in `app'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/server.rb:353:in `wrapped_app'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/rack-2.0.1/lib/rack/server.rb:282:in `start'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/commands/server.rb:79:in `start'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/commands/commands_tasks.rb:90:in `block in server'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/commands/commands_tasks.rb:85:in `tap'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/commands/commands_tasks.rb:85:in `server'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/commands/commands_tasks.rb:49:in `run_command!'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0/lib/rails/commands.rb:18:in `<top (required)>'
	from /Users/alfredo/Documents/workspace/zulu-rails/bin/rails:9:in `require'
	from /Users/alfredo/Documents/workspace/zulu-rails/bin/rails:9:in `<top (required)>'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/spring-1.7.2/lib/spring/client/rails.rb:28:in `load'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/spring-1.7.2/lib/spring/client/rails.rb:28:in `call'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/spring-1.7.2/lib/spring/client/command.rb:7:in `call'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/spring-1.7.2/lib/spring/client.rb:30:in `run'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/spring-1.7.2/bin/spring:49:in `<top (required)>'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/spring-1.7.2/lib/spring/binstub.rb:11:in `load'
	from /Users/alfredo/.rvm/gems/ruby-2.3.1/gems/spring-1.7.2/lib/spring/binstub.rb:11:in `<top (required)>'
	from /Users/alfredo/Documents/workspace/zulu-rails/bin/spring:13:in `require'
	from /Users/alfredo/Documents/workspace/zulu-rails/bin/spring:13:in `<top (required)>'
	from bin/rails:3:in `load'
	from bin/rails:3:in `<main>'
```

The part of the program which does something naughty is line 21 of the engine:

```
      config.middleware.insert_before 0, rack_cors_class do
        allow do
          hostnames = ['localhost:4200', 'app.forestadmin.com',
                       'www.forestadmin.com']
          hostnames += ENV['CORS_ORIGINS'].split(',') if ENV['CORS_ORIGINS']

          origins hostnames
          resource '*', headers: :any, methods: :any
        end
```

I've set my `CORS_ORIGINS` env var: `export CORS_ORIGINS=zulu-rails.herokuapp.com` (name of my app on heroku. I've set it up that way on my machine and my heroku dyno and I still run into the same issue on both.

I've managed to successfully work around the issue by removing the cors configuration call from the engine and doing cors initialization instead from an initialier under `/config/initializers/cors.rb`, but of course I understand this is *not sensible*.

I actually have the time to look into this and contribute if you guys don't mind, but it's 2 AM over here and I gotta rest a bit xD. Will check in back again in the morning.

**Comment by nullset2** [comment]
By the way, this is Rails 5 on `api-mode`:

```
rails middleware
use Rack::Sendfile
use ActionDispatch::Static
use ActionDispatch::Executor
use ActiveSupport::Cache::Strategy::LocalCache::Middleware
use Rack::Runtime
use ActionDispatch::RequestId
use Rails::Rack::Logger
use ActionDispatch::ShowExceptions
use ActionDispatch::DebugExceptions
use ActionDispatch::RemoteIp
use ActionDispatch::Reloader
use ActionDispatch::Callbacks
use ActiveRecord::Migration::CheckPending
use Rack::Head
use Rack::ConditionalGet
use Rack::ETag
use Warden::Manager
use Apipie::Extractor::Recorder::Middleware
use Apipie::StaticDispatcher
run ZuluRails::Application.routes
```

**Comment by arnaudbesnier** [comment]
Hi Alfredo!

Thank you for the report. It seems to be a real issue and this is _sensible_ to us.
We'll have a look into your issue today.

By the way you have an option to deactivate the Forest CORS configuration. You just have to set:
```ruby
ForestLiana.inject_forest_cors = false
```
in the `config/initializers/forest_liana.rb` of your project.

**Comment by nullset2** [comment]
Amazing, setting that option to false allows me to launch my web server on production all _hunky dory_.

I'll use an initializer to initialize the cors middleware for now instead --that is a _sensible_ workaround for me.

Cheers guys!

**Comment by arnaudbesnier** [comment]
Hey Alfredo!

We are still looking to reproduce the issue.
Could you kindly provide us your `Gemfile.lock` to let us see the versions of the gems you use.

One other info that could be great to see is your `config/environment/production.rb` file to understand what it specifically done in production where the error happens.

**Comment by GabKlein** [comment]
Same here but with rails 4.2.7.1

```
GIT
  remote: git://github.com/GabKlein/ahoy.git
  revision: 1032532d4e9fe4ba2992f676cebcc9bea3ada429
  specs:
    ahoy_matey (1.4.0)
      addressable
      browser (~> 2.0)
      geocoder
      rack-attack
      railties
      referer-parser (>= 0.3.0)
      request_store
      safely_block (>= 0.1.1)
      user_agent_parser
      uuidtools

GIT
  remote: git://github.com/GabKlein/forest-rails.git
  revision: 26573f6d4dbefff474022d808f9b36a8f0985b85
  branch: devel
  specs:
    forest_liana (1.4.3)
      arel-helpers
      bcrypt
      groupdate
      jsonapi-serializers (>= 0.14.0)
      jwt
      rack-cors
      rails (>= 3.0)
      useragent

GIT
  remote: git://github.com/svenfuchs/rails-i18n.git
  revision: dc6e3ec7e59e0a1ea2933bf6b4f107a9df419e26
  branch: rails-4-x
  specs:
    rails-i18n (4.0.9)
      i18n (~> 0.7)
      railties (~> 4.0)

GEM
  remote: https://rubygems.org/
  specs:
    actionmailer (4.2.7.1)
      actionpack (= 4.2.7.1)
      actionview (= 4.2.7.1)
      activejob (= 4.2.7.1)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 1.0, >= 1.0.5)
    actionpack (4.2.7.1)
      actionview (= 4.2.7.1)
      activesupport (= 4.2.7.1)
      rack (~> 1.6)
      rack-test (~> 0.6.2)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (4.2.7.1)
      activesupport (= 4.2.7.1)
      builder (~> 3.1)
      erubis (~> 2.7.0)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    active_model_serializers (0.10.2)
      actionpack (>= 4.1, < 6)
      activemodel (>= 4.1, < 6)
      jsonapi (~> 0.1.1.beta2)
      railties (>= 4.1, < 6)
    activejob (4.2.7.1)
      activesupport (= 4.2.7.1)
      globalid (>= 0.3.0)
    activemodel (4.2.7.1)
      activesupport (= 4.2.7.1)
      builder (~> 3.1)
    activerecord (4.2.7.1)
      activemodel (= 4.2.7.1)
      activesupport (= 4.2.7.1)
      arel (~> 6.0)
    activesupport (4.2.7.1)
      i18n (~> 0.7)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    addressable (2.5.0)
      public_suffix (~> 2.0, >= 2.0.2)
    ansi (1.5.0)
    apipie-rails (0.3.7)
      json
    arel (6.0.3)
    arel-helpers (2.3.0)
      activerecord (>= 3.1.0, < 6)
    ast (2.3.0)
    autoprefixer-rails (6.5.3)
      execjs
    awesome_print (1.7.0)
    aws-sdk (2.6.23)
      aws-sdk-resources (= 2.6.23)
    aws-sdk-core (2.6.23)
      aws-sigv4 (~> 1.0)
      jmespath (~> 1.0)
    aws-sdk-resources (2.6.23)
      aws-sdk-core (= 2.6.23)
    aws-sigv4 (1.0.0)
    bcrypt (3.1.11)
    bootstrap-sass (3.3.7)
      autoprefixer-rails (>= 5.2.1)
      sass (>= 3.3.4)
    brakeman (3.4.1)
    browser (2.3.0)
    builder (3.2.2)
    byebug (9.0.6)
    carrierwave (0.11.2)
      activemodel (>= 3.2.0)
      activesupport (>= 3.2.0)
      json (>= 1.7)
      mime-types (>= 1.16)
      mimemagic (>= 0.3.0)
    carrierwave-aws (1.0.2)
      aws-sdk (~> 2.0)
      carrierwave (>= 0.7, < 2.0)
    chewy (0.8.4)
      activesupport (>= 3.2)
      elasticsearch (>= 1.0.0)
    choice (0.2.0)
    chunky_png (1.3.4)
    ci_reporter (2.0.0)
      builder (>= 2.1.2)
    ci_reporter_rspec (1.0.0)
      ci_reporter (~> 2.0)
      rspec (>= 2.14, < 4)
    clockwork (2.0.0)
      activesupport
      tzinfo
    coderay (1.1.1)
    concurrent-ruby (1.0.2)
    connection_pool (2.2.1)
    contracts (0.14.0)
    crack (0.4.3)
      safe_yaml (~> 1.0.0)
    dalli (2.7.6)
    database_cleaner (1.5.3)
    diff-lcs (1.2.5)
    docile (1.1.5)
    domain_name (0.5.20161021)
      unf (>= 0.0.5, < 1.0.0)
    dotenv (2.1.1)
    dragonfly (1.1.1)
      addressable (~> 2.3)
      multi_json (~> 1.0)
      rack (>= 1.3)
    elasticsearch (5.0.0)
      elasticsearch-api (= 5.0.0)
      elasticsearch-transport (= 5.0.0)
    elasticsearch-api (5.0.0)
      multi_json
    elasticsearch-extensions (0.0.22)
      ansi
      ruby-prof
    elasticsearch-transport (5.0.0)
      faraday
      multi_json
    email_spec (2.1.0)
      htmlentities (~> 4.3.3)
      launchy (~> 2.1)
      mail (~> 2.6.3)
    errbase (0.0.3)
    erubis (2.7.0)
    eu_central_bank (1.0.1)
      money (~> 6.7.0)
      nokogiri (~> 1.6.3)
    execjs (2.7.0)
    factory_girl (4.7.0)
      activesupport (>= 3.0.0)
    factory_girl_rails (4.7.0)
      factory_girl (~> 4.7.0)
      railties (>= 3.0.0)
    faker (1.6.6)
      i18n (~> 0.5)
    fakeredis (0.6.0)
      redis (~> 3.2)
    faraday (0.10.0)
      multipart-post (>= 1.2, < 3)
    friendly_id (5.1.0)
      activerecord (>= 4.0.0)
    geocoder (1.4.0)
    globalid (0.3.7)
      activesupport (>= 4.1.0)
    groupdate (3.1.1)
      activesupport (>= 3)
    hashdiff (0.3.0)
    htmlentities (4.3.4)
    http-cookie (1.0.3)
      domain_name (~> 0.5)
    http_accept_language (2.1.0)
    httpclient (2.8.2.4)
    i18n (0.7.0)
    i18n-country-translations (1.2.4)
      i18n (~> 0.5)
      railties (>= 4.0)
    iso-iban (0.1.3)
    jmespath (1.3.1)
    json (1.8.3)
    jsonapi (0.1.1.beta6)
      jsonapi-parser (= 0.1.1.beta3)
      jsonapi-renderer (= 0.1.1.beta1)
    jsonapi-parser (0.1.1.beta3)
    jsonapi-renderer (0.1.1.beta1)
    jsonapi-serializers (0.16.1)
      activesupport
    jwt (1.5.6)
    kaminari (0.17.0)
      actionpack (>= 3.0.0)
      activesupport (>= 3.0.0)
    launchy (2.4.3)
      addressable (~> 2.3)
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    mail (2.6.4)
      mime-types (>= 1.16, < 4)
    method_source (0.8.2)
    mime-types (2.99.3)
    mimemagic (0.3.2)
    mini_magick (4.5.1)
    mini_portile2 (2.1.0)
    minitest (5.9.1)
    monetize (1.4.0)
      money (~> 6.7)
    money (6.7.1)
      i18n (>= 0.6.4, <= 0.7.0)
      sixarm_ruby_unaccent (>= 1.1.1, < 2)
    money-rails (1.7.0)
      activesupport (>= 3.0)
      monetize (~> 1.4.0)
      money (~> 6.7)
      railties (>= 3.0)
    multi_json (1.12.1)
    multipart-post (2.0.0)
    netrc (0.11.0)
    newrelic_rpm (3.17.1.326)
    nokogiri (1.6.8.1)
      mini_portile2 (~> 2.1.0)
    oj (2.17.5)
    onesky-rails (1.4.0)
      i18n (>= 0.5.0)
      onesky-ruby (~> 1.0.0)
    onesky-ruby (1.0.1)
      rest-client (~> 1.8)
    paper_trail (5.2.2)
      activerecord (>= 3.0, < 6.0)
      request_store (~> 1.1)
    paranoia (2.2.0)
      activerecord (>= 4.0, < 5.1)
    parser (2.3.1.4)
      ast (~> 2.2)
    pg (0.19.0)
    powerpack (0.1.1)
    pry (0.10.4)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
    pry-byebug (3.4.0)
      byebug (~> 9.0)
      pry (~> 0.10)
    pry-doc (0.9.0)
      pry (~> 0.9)
      yard (~> 0.8)
    pry-rails (0.3.4)
      pry (>= 0.9.10)
    public_suffix (2.0.4)
    puma (3.6.0)
    pundit (1.1.0)
      activesupport (>= 3.0.0)
    rack (1.6.5)
    rack-attack (5.0.1)
      rack
    rack-cors (0.4.0)
    rack-protection (1.5.3)
      rack
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (4.2.7.1)
      actionmailer (= 4.2.7.1)
      actionpack (= 4.2.7.1)
      actionview (= 4.2.7.1)
      activejob (= 4.2.7.1)
      activemodel (= 4.2.7.1)
      activerecord (= 4.2.7.1)
      activesupport (= 4.2.7.1)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.2.7.1)
      sprockets-rails
    rails-api (0.4.0)
      actionpack (>= 3.2.11)
      railties (>= 3.2.11)
    rails-deprecated_sanitizer (1.0.3)
      activesupport (>= 4.2.0.alpha)
    rails-dom-testing (1.0.7)
      activesupport (>= 4.2.0.beta, < 5.0)
      nokogiri (~> 1.6.0)
      rails-deprecated_sanitizer (>= 1.0.1)
    rails-erd (1.5.0)
      activerecord (>= 3.2)
      activesupport (>= 3.2)
      choice (~> 0.2.0)
      ruby-graphviz (~> 1.2)
    rails-html-sanitizer (1.0.3)
      loofah (~> 2.0)
    rails_12factor (0.0.3)
      rails_serve_static_assets
      rails_stdout_logging
    rails_serve_static_assets (0.0.5)
    rails_stdout_logging (0.0.5)
    railties (4.2.7.1)
      actionpack (= 4.2.7.1)
      activesupport (= 4.2.7.1)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rainbow (2.1.0)
    rake (11.3.0)
    redis (3.3.2)
    redis-namespace (1.5.2)
      redis (~> 3.0, >= 3.0.4)
    referer-parser (0.3.0)
    request_store (1.3.1)
    rest-client (1.8.0)
      http-cookie (>= 1.0.2, < 2.0)
      mime-types (>= 1.16, < 3.0)
      netrc (~> 0.7)
    rotp (3.3.0)
    rqrcode (0.10.1)
      chunky_png (~> 1.0)
    rspec (3.5.0)
      rspec-core (~> 3.5.0)
      rspec-expectations (~> 3.5.0)
      rspec-mocks (~> 3.5.0)
    rspec-activejob (0.6.1)
      activejob (>= 4.2)
      rspec-mocks
    rspec-core (3.5.4)
      rspec-support (~> 3.5.0)
    rspec-expectations (3.5.0)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.5.0)
    rspec-mocks (3.5.0)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.5.0)
    rspec-rails (3.5.2)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      railties (>= 3.0)
      rspec-core (~> 3.5.0)
      rspec-expectations (~> 3.5.0)
      rspec-mocks (~> 3.5.0)
      rspec-support (~> 3.5.0)
    rspec-set (0.1.3)
    rspec-support (3.5.0)
    rubocop (0.45.0)
      parser (>= 2.3.1.1, < 3.0)
      powerpack (~> 0.1)
      rainbow (>= 1.99.1, < 3.0)
      ruby-progressbar (~> 1.7)
      unicode-display_width (~> 1.0, >= 1.0.1)
    ruby-graphviz (1.2.2)
    ruby-prof (0.16.2)
    ruby-progressbar (1.8.1)
    rubyzip (1.2.0)
    safe_yaml (1.0.4)
    safely_block (0.1.1)
      errbase
    sass (3.4.22)
    sass-rails (5.0.6)
      railties (>= 4.0.0, < 6)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (>= 1.1, < 3)
    sequenced (3.1.1)
      activerecord (>= 3.0)
      activesupport (>= 3.0)
    sidekiq (4.2.6)
      concurrent-ruby (~> 1.0)
      connection_pool (~> 2.2, >= 2.2.0)
      rack-protection (>= 1.5.0)
      redis (~> 3.2, >= 3.2.1)
    sidekiq-unique-jobs (4.0.18)
      sidekiq (>= 2.6)
      thor
    simplecov (0.12.0)
      docile (~> 1.1.0)
      json (>= 1.8, < 3)
      simplecov-html (~> 0.10.0)
    simplecov-html (0.10.0)
    sinatra (1.4.7)
      rack (~> 1.5)
      rack-protection (~> 1.4)
      tilt (>= 1.3, < 3)
    sixarm_ruby_unaccent (1.1.1)
    skylight (1.0.1)
      activesupport (>= 3.0.0)
    slim (3.0.7)
      temple (~> 0.7.6)
      tilt (>= 1.3.3, < 2.1)
    slop (3.6.0)
    sprockets (3.7.0)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.0)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    sucker_punch (2.0.2)
      concurrent-ruby (~> 1.0.0)
    temple (0.7.7)
    thor (0.19.1)
    thread_safe (0.3.5)
    tilt (2.0.5)
    timecop (0.8.1)
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.7.2)
    unicode-display_width (1.1.1)
    user_agent_parser (2.3.0)
    useragent (0.16.8)
    uuidtools (2.1.5)
    webmock (2.1.0)
      addressable (>= 2.3.6)
      crack (>= 0.3.2)
      hashdiff
    wicked_pdf (1.1.0)
    wisper (2.0.0.rc1)
    wisper-rspec (0.0.2)
    wkhtmltopdf-binary (0.12.3.1)
    yard (0.9.5)

PLATFORMS
  ruby

DEPENDENCIES
  active_model_serializers (~> 0.10.0)
  ahoy_matey!
  apipie-rails
  awesome_print
  bcrypt
  bootstrap-sass
  brakeman
  carrierwave
  carrierwave-aws
  chewy
  chunky_png (= 1.3.4)
  ci_reporter
  ci_reporter_rspec
  clockwork
  connection_pool
  contracts
  dalli
  database_cleaner
  dotenv
  dragonfly
  elasticsearch-extensions
  email_spec
  eu_central_bank
  factory_girl_rails
  faker
  fakeredis
  faraday
  forest_liana!
  friendly_id
  http_accept_language
  httpclient
  i18n-country-translations
  iso-iban
  jwt
  kaminari
  mini_magick
  money-rails
  multi_json
  newrelic_rpm
  oj
  onesky-rails
  paper_trail
  paranoia
  pg
  pry-byebug
  pry-doc
  pry-rails
  puma
  pundit
  rack-attack
  rack-cors
  rails (= 4.2.7.1)
  rails-api
  rails-erd
  rails-i18n!
  rails_12factor
  redis
  redis-namespace
  rotp
  rqrcode
  rspec-activejob
  rspec-mocks
  rspec-rails
  rspec-set
  rubocop
  rubyzip
  sass-rails
  sequenced
  sidekiq
  sidekiq-unique-jobs
  simplecov
  sinatra
  skylight
  slim
  sprockets-rails
  sucker_punch (~> 2.0)
  timecop
  webmock
  wicked_pdf
  wisper (= 2.0.0.rc1)
  wisper-rspec
  wkhtmltopdf-binary

RUBY VERSION
   ruby 2.3.1p112

BUNDLED WITH
   1.12.5
```

**Comment by arnaudbesnier** [comment]
@GabKlein we have a Pull Request currently under review to fix this regression: https://github.com/ForestAdmin/forest-rails/pull/73

**Comment by arnaudbesnier** [comment]
@nullset2 @GabKlein The fix has been released, you can update the gem to version 1.4.4!

Don't hesitate to reopen the issue if necessary.
Thanks again for your feedback.
